benchmarks/icfp15/README.md

```diff
@@ -0,0 +1,4 @@
+# Examples from _Bounded Refinement Types_
+
+The source code of the examples from Vazou–Bakst–Jhala's ICFP'15 paper is now
+in `tests/benchmarks/icfp15`.
```

Comment:
This works! Probably there is some way to make it a link.
```
[tests/benchmarks/icfp15](../../tests/benchmarks/icfp15)
```
or
```
[tests/benchmarks/icfp15](https://github.com/ucsd-progsys/liquidhaskell/tree/develop/tests/benchmarks/icfp15)
```

Reply:
Let it be so.  I am definitely going for the first one (relative path) because if one already is working on a local copy, the link should still point to within the local copy.